save-demo-agent/src/nativeMain/kotlin/com/saveourtool/save/demo/agent/utils/HttpUtils.kt

```diff
@@ -30,7 +31,7 @@ import okio.use
 import kotlin.native.concurrent.AtomicLong
 
 private val httpClient = HttpClient(CIO) {
-    install(ContentNegotiation)
+    install(ContentNegotiation) { json() }
```

Comment:
You just should have copy-pasted example from ktor readme, what should have gone wrong…

Reply:
I didn't copy-paste...